backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -111,6 +111,7 @@
 import org.springframework.boot.info.GitProperties;
 import org.springframework.data.util.Pair;
 import org.springframework.util.CollectionUtils;
+import org.thymeleaf.util.ListUtils;
```

Comment:
Is there a reason why thymeleaf's ListUtils was used over something like Apache commons CollectionUtils?

Reply:
no reason, I'll change it